src/sentry/event_manager.py

```diff
@@ -849,14 +855,22 @@ def save(self, project, raw=False):
                 timestamp=event.datetime,
                 environment_id=environment.id,
             )
-
-        if is_new and release:
-            buffer.incr(
-                ReleaseProject, {'new_groups': 1}, {
-                    'release_id': release.id,
-                    'project_id': project.id,
-                }
-            )
+        if release:
+            if is_new:
+                buffer.incr(
+                    ReleaseProject, {'new_groups': 1}, {
+                        'release_id': release.id,
+                        'project_id': project.id,
+                    }
+                )
+            if is_new_group_environment:
+                buffer.incr(
+                    ReleaseProjectEnvironment, columns={'new_issues_count': 1}, filters={
+                        'project': project,
```

Comment:
This needs to explicitly use `project_id` and friends. We shouldn't pass the full objects through.

And I'd also argue that for consistency, we don't ned to use `columns` and `filters` kwargs. Just to conform with how it's used elsewhere.

Reply:
@mattrobenolt so the model has flexibleforiegnkeys, does that automatically make 
```
'project_id':project_id
```
A valid statement?

Reply:
@mattrobenolt I used them just to know what they were; I didn't understand what was going on at first glance. I'd argue that this is clearer. But no I hadn't seen it used elsewhere. It's just a style issue, so I will change it since it bothers you.

Reply:
You'd have to use `'project_id': project.id` in this context, but yes, it'll just work.